src/openrct2/paint/tile_element/Surface.cpp

```diff
@@ -149,7 +149,7 @@ static constexpr const uint8 byte_97B55D[] = {
     1, 5, 1, 3, 2, 3, 1, 5, 0
 };
 
-static constexpr const uint8 stru_97B570[][2] = {
+static constexpr const uint8 _tunnelHeights[TUNNEL_TYPE_COUNT][2] = {
```

Comment:
(Late comment, I'm a little bit behind :P) but why are there 2 values for each entry?

Reply:
I'm not 100% sure, but I think the first is the (base) height and the second is the bound box height, judging from the code.